recipes/whisper-cpp/all/conanfile.py

```diff
@@ -0,0 +1,158 @@
+                       "sanitize_address": False, "sanitize_undefined": False, "with_sdl2": False,
+                       "no_avx": False, "no_avx2": False, "no_fma": False, "no_f16c": False,
+                       "no_accelerate": False, "with_coreml": False, "coreml_allow_fallback": False,
+                       "with_blas": False}
+
+    _cmake = None
+
+    @property
+    def _min_cppstd(self):
+        return "14"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "14": {
+                "gcc": "9",
+                "clang": "5",
+                "apple-clang": "10",
+                "Visual Studio": "15",
+                "msvc": "191",
+            },
+        }.get(self._min_cppstd, {})
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
+        if is_apple_os(self):
+            del self.options.with_blas
+        else:
+            del self.options.no_accelerate
+            del self.options.with_coreml
+            del self.options.coreml_allow_fallback
+
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+        if is_apple_os(self):
+            if not self.options.with_coreml:
+                self.options.rm_safe("coreml_allow_fallback")
+
+    def validate(self):
+        if self.settings.compiler.cppstd:
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(
+                f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+            )
+
+    def requirements(self):
+        if self.options.with_sdl2:
+            self.requires("sdl/[>=2.0.0]")
+
+        if not is_apple_os(self):
+            if self.options.with_blas:
+                self.requires("openblas/[>=0.3.7]")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["WHISPER_BUILD_TESTS"] = False
+        tc.variables["WHISPER_BUILD_EXAMPLES"] = False
+
+        if self.options.shared:
+            tc.variables["BUILD_SHARED_LIBS"] = True
+        if self.options.sanitize_thread:
+            tc.variables["WHISPER_SANITIZE_THREAD"] = True
+        if self.options.sanitize_address:
+            tc.variables["WHISPER_SANITIZE_ADDRESS"] = True
+        if self.options.sanitize_undefined:
+            tc.variables["WHISPER_SANITIZE_UNDEFINED"] = True
+        if self.options.with_sdl2:
+            tc.variables["WHISPER_SDL2"] = True
```

Comment:
Just checking. Did you make sure that the dependencies are correctly used from Conan?

Reply:
I did a quick check and the answer was that I wasn't using them correctly, which has since been addressed. I ended up removing SDL2 as I can't figure out what the library actually does with it, and correctly linked up OpenBLAS for both the stable and beta versions.